README.md

```diff
@@ -1 +1,27 @@
 # Recipe onboarding app
+CRUD API with Django and DRF for the onboarding.
+
+## Starting the app
+```shell
+docker compose up
+```
```

Comment:
Nice readme btw ❤️ 

I think it's just missing the migration, at least it was necessary for me to run the app
```shell
docker compose run --rm app sh -c "python manage.py migrate"
```

Reply:
Updated the up command to run the migrations as well.